lib/resource_dsl/acts_as_audit.rb

```diff
@@ -34,7 +34,7 @@ def show
       end
 
       action_item :history, only: %i[show edit] do
-        if authorized?(:history)
+        if authorized?(:history) && is_versioning_enabled_for_model?(resource)
```

Comment:
try to use `resource_class` instead, it will allow you to skip checking decoration
`is_versioning_enabled_for_model_class?(resource_class)`